### DISCUSSION TITLE
setAuth(access_token) does not return or set user within sessions

### DISCUSSION THREAD

**Discussion by tejaskamath** [supabase/supabase#7271] [Questions]
So our remix application relies on setting access token to the supabase client. Though the access token works and authenticates requests, this does not set the user to the session  or return a user object and hence we rely on supabase.auth.api.getUser(access_token) to retrieve the user information. 

I noticed in the docs however that it does return a user object, which is returning null in our case.
https://supabase.com/docs/reference/javascript/auth-setauth

Is it possible to set access tokens and obtain users in a single call?

**Comment by GaryAustin1** [comment]
Setauth does not return user.  It does not make a server call. Issue on docs here: https://github.com/supabase/gotrue-js/issues/247
